content/guides/references/migration-guide.md

```diff
@@ -83,6 +83,158 @@ on('<event>', (arg1, arg2) => {
 
 CONTENT_TBD
 
+## Migrating `cypress-file-upload` to [`.selectFile()`](/api/commands/selectfile)
+
+Selecting files with input elements or dropping them over the page is available
+in Cypress 9.3. Read the [`.selectFile()` API docs](/api/commands/selectfile)
+for more information on how this works and how to use it. This guide details how
+to change your test code to migrate from
+[`cypress-file-upload`](https://github.com/abramenal/cypress-file-upload) to
+`.selectFile()`.
+
+#### Quick guide
+
+The argument signature is different for Cypress' builtin `.selectFile()` command
+than the `.attachFile` command the `cypress-file-upload` plugin provided. You
+can follow the steps below for each argument in order to migrate:
+
+In the first argument:
+
+- `filePath`: Prefix value with `cypress/fixtures/`. Rename to `contents`.
+- `fileContent`: Use `Buffer.from()` rather than `Cypress.Blob` methods. Rename
+  to `contents`.
+- `encoding`: Remove. No longer needed due to improved binary file handling in
+  Cypress 9.0.
+- `mimeType`: Remove. No longer needed due to improved binary file handling in
+  Cypress 9.0.
+
+In the second argument:
+
+- `subjectType`: Rename to `action`. Move from `drag-n-drop` to `drag-drop` or
+  from `input` to `select`.
+- `allowEmpty`: Remove. `.selectFile()` does not check the length of a file read
+  from disk, only its existence.
```

Comment:
Could you change this section to something along these lines? I want to be as clear as possible with the language:

When the first argument is a string filepath (or array of string filepaths):

- Prefix the existing filepath(s) with `cypress/fixtures/`.

When the first argument is an object:

- `filePath`: Rename this property to `contents`. Prefix the existing value with `cypress/fixtures/`.
- `fileContent`: Rename this property to `contents`. Use `Buffer.from()` rather than `Cypress.Blob` methods. 
- `encoding`: Remove this property. No longer needed due to improved binary file handling in Cypress 9.0.
- `mimeType`: Remove this property. No longer needed due to improved binary file handling in Cypress 9.0.

The second argument:

- `subjectType`: Rename this property to `action`. Change the existing value from `drag-n-drop` to `drag-drop`, or from `input` to `select`.
- `allowEmpty`: Remove this property. The `.selectFile()` command does not check the length of a file read from disk, only its existence.

Reply:
Good suggestions, applied.